### ISSUE TITLE
Web Content Accessibility Guidelines (WCAG) 2.2 2023-01-30 > 2023-02-24

### ISSUE THREAD

**Issue by michael-n-cooper** [w3ctag/design-reviews#811]
### Name of your specification

Web Content Accessibility Guidelines (WCAG) 2.2

### URL of your specification

https://www.w3.org/TR/2023/CRD-WCAG22-20230125/

### When does the review need to be finished?

2023-02-24

### What has changed since any previous review?

https://www.w3.org/TR/WCAG22/#substantive-changes-since-the-6-september-2022-candidate-recommendation

### Where and how should the i18n WG raise issues?

https://github.com/w3c/wcag/issues/

**Comment by torgo** [comment]
Hi @michael-n-cooper thanks for this! We are just setting this up for discussion in our virtual face-to-face. Can you please send an [explainer](https://w3ctag.org/explainers/) to give us a little more context on this review?

**Comment by michael-n-cooper** [comment]
Hi @torgo WCAG 2.2 doesn't have a formal explainer, it's been continuing the old WCAG 2.x work. Explanatory material is in an [Overview](https://www.w3.org/WAI/standards-guidelines/wcag/) and the [Understanding WCAG 2.2](https://www.w3.org/WAI/WCAG22/Understanding/) materials. This horizontal review is focused on [changes since the previous CR](https://www.w3.org/TR/2023/CRD-WCAG22-20230125/#substantive-changes-since-the-6-september-2022-candidate-recommendation).

**Comment by torgo** [comment]
Thanks @michael-n-cooper that's really helpful.

**Comment by torgo** [comment]
Hi @michael-n-cooper - thanks again for bringing this to our attention as part of wide review.  We've reviewed and we're happy to see this go forward.   Can you clarify regarding the parsing issue (4.1.1) - just for our benefit so we understand why AT is no longer parsing HTML? Are you confident that there aren't any remaining edge cases where that would still be needed?

**Comment by torgo** [comment]
@michael-n-cooper as discussed in today's TAG call, we're happy to close this as *satisfied* - thanks again for sending this our way. If you can let us know some more detail on the issue I asked about above that would be great. Thanks for flying TAG.
